cmds/app.c

```diff
@@ -30,17 +30,23 @@ static void cmd_appInfo(void)
 static int cmd_cpphfs2map(handler_t handler, const char *imap)
 {
 	int res;
-	addr_t offs = 0;
+	addr_t top, offs = 0;
 	u8 buff[SIZE_MSG_BUFF];
 
+	if ((res = syspage_getMapTop(imap, &top)) < 0)
```

Comment:
`syspage_getMapTop` is invoked here:
https://github.com/phoenix-rtos/plo/blob/baefaa9deb107d483d23bf48db1164819baae9f0/cmds/app.c#L82-L85
I propose to pass top value as argument to `cmd_cpphfs2map`.

Reply:
Done.